MarkdownToLatex/MarkdownToLatex.Test/TestMdToTex.cs

```diff
@@ -150,5 +154,41 @@ public void TestConvert()
             //assert
             Assert.Equal(expTex, result);
 		}
+
+        [Fact]
+        public void TestParseInputPath(){
+            //arrange
+            string path1 = @"test_files/testAll.md";
+            string path2 = @"test_files/latex/test1.tex";
+            string path3 = @"test_files/nonexistantfile.md";
+
+            //act
+            Action parse1 = new Action(() => {string parsedPath1 = MdToTex.parseInputPath(path1);});
+            Action parse2 = new Action(() => {string parsedPath2 = MdToTex.parseInputPath(path2);});
```

Comment:
Die Datei existiert bei mir noch nicht, also gibt es eine `FileNotFoundException`. Commite die mal mit oder besser, erstell die im Code.

Reply:
Stimmt, wird erledigt

Reply:
Ist die jetzt erstellt? Ich seh die irgendwie nicht.  
Bzw. ist das Problem irgendwie anders gelöst und ich kapier es nur nicht?

Reply:
Ich habe eigentlich in 6810871 dort ein `File.Create(path2).Close()` hinzugefügt, um die Datei zu erstellen und sofort wieder zu schließen (Damit der andere Test, der auf die Datei zugreift nicht rummeckert). Irgendwie hat GitHub das aber nicht richtig angezeigt. Ich hab jetzt mal in 00148f9 danach ne Leerzeile eingefügt, vielleicht zeigt er es jetzt an